sdk/apimanagement/Azure.ResourceManager.ApiManagement/src/Customized/Models/ApiManagementContentType.cs

```diff
@@ -29,15 +30,27 @@ public ApiManagementContentTypeData()
         /// <param name="description"> Content type description. </param>
         /// <param name="schema"> Content type schema. </param>
         /// <param name="version"> Content type version. </param>
-        internal ApiManagementContentTypeData(ResourceIdentifier id, string name, ResourceType resourceType, SystemData systemData, string contentTypeIdentifier, string contentTypeName, string description, BinaryData schema, string version) : base(id, name, resourceType, systemData)
+        internal ApiManagementContentType(string id, string name, ResourceType resourceType, ResourceManager.Models.SystemData systemData, string contentTypeIdentifier, string contentTypeName, string description, BinaryData schema, string version)
         {
+            ContentTypeId = id;
+            Name = name;
+            ResourceType = resourceType;
+            SystemData = systemData;
             ContentTypeIdentifier = contentTypeIdentifier;
             ContentTypeName = contentTypeName;
             Description = description;
             Schema = schema;
             Version = version;
         }
 
+        /// <summary> Content Type ID. </summary>
+        public string ContentTypeId { get; }
+        /// <summary> The name of the resource. </summary>
+        public string Name { get; }
+        /// <summary> The type of the resource. E.g. &quot;Microsoft.Compute/virtualMachines&quot; or &quot;Microsoft.Storage/storageAccounts&quot;. </summary>
+        public ResourceType ResourceType { get; }
+        /// <summary> Azure Resource Manager metadata containing createdBy and modifiedBy information. </summary>
+        public ResourceManager.Models.SystemData SystemData { get; }
```

Comment:
Just emphasizing this for you to double check - does it really have `systemData`?
The systemData might come from our ResourceData

Reply:
Thanks for the reminder, double checked and removed this property.